idd/Energy+.idd.in

```diff
@@ -57223,17 +57223,41 @@ Chiller:Electric:ReformulatedEIR,
        \note curve = a + b*CWS + c*CWS**2 + d*LCT + e*LCT**2 + f*CWS*LCT
        \note CWS = supply (leaving) chilled water temperature(C)
        \note LCT = leaving condenser fluid temperature(C)
-  A4 , \field Electric Input to Cooling Output Ratio Function of Part Load Ratio Curve Name
+  A4, \field Electric Input to Cooling Output Ratio Function of Part Load Ratio Curve Type
+       \note Two part-load curves are available. 
+       \note The previous one uses the leaving condenser water temperature.
+       \note The new one uses the normalized lift. Lift is the temperature difference between the
+       \note leaving condenser water temperature and the leaving evaporator water temperature. 
+       \note Two new terms are included in the new curve to improve model accuracy when the leaving chilled 
+       \note water temperature deviates from the design conditions especially at high part-load conditions.
+       \required-field
+       \type choice
+       \key UseLeavingCondenserWaterTemperature
+       \key UseLift
+       \default UseLeavingCondenserWaterTemperature
+  A5 , \field Electric Input to Cooling Output Ratio Function of Part Load Ratio Curve Name
        \note Electric Input Ratio (EIR) as a function of Part Load Ratio (PLR)
        \note EIR = 1/COP
        \required-field
        \type object-list
        \object-list BicubicCurves
-       \object-list BiVariateTables
+       \object-list ChillerPartLoadWithLiftCurves
+       \note two types of part-load curves are available: BicubicCurves or ChillerPartLoadWithLiftCurves
+       \note Type 1: BicubicCurves; 
+       \note Calculated based on LCT and PLR       
        \note curve = a + b*LCT + c*LCT**2 + d*PLR + e*PLR**2 + f*LCT*PLR + g*0 + h*PLR**3
        \note + i*0 + j*0
        \note PLR = part load ratio (cooling load/steady state capacity)
        \note LCT = leaving condenser fluid temperature(C)
+       \note Type 2: ChillerPartLoadWithLiftCurves 
+       \note Calculated based on dT*, Tdev* and PLR
+       \note curve = a + b*(dT*) + c*(dT*)**2 + d*PLR + e*PLR**2 + f*(dT*)*PLR + g*(dT*)**3
+       \note + h*PLR**3 + i*(dT*)**2*PLR + j*(dT*)*PLR**2 + k*(dT*)**2*PLR**2 + l*(Tdev*)*PLR**3
+       \note Tdev = Leaving Chilled Water Temperature � Reference Chilled Water Temperature
```

Comment:
@rongpengzhang There is a special character in this line - shows as a box on github.